web/src/javascript/game.js

```diff
@@ -0,0 +1,18 @@
+import {
+  drawCellsOnMap,
+  drawMap,
+  drawMiniMap,
+} from "./drawMap";
+
+function extractPointsFromState(gameState) {
+  return [].concat.apply(...gameState.players.map((p) => p.cells));
+}
+
+export function draw(gameState) {
+  drawCellsOnMap(extractPointsFromState(gameState));
```

Comment:
drawCellsOnMap needs an array of cell. On row 8, there's the `...` in front of `gameState.players.map`. I don't understand what the line 8 really does...